starter-code/javascript/chronometer.js

```diff
@@ -1,40 +1,55 @@
 // Constructor
-// function Chronometer() {
-
-// }
-
-// Chronometer.prototype.startClick = function () {
-
-// };
-
-// Chronometer.prototype.setMinutes = function () {
+function Chronometer() {
+  this.currentTime = 0;
+  this.intervalId = 0;
+}
+
+Chronometer.prototype.startClick = function () {
+  var that = this;
+  this.intervalId = setInterval(function(){
+    parseInt(that.currentTime++);
+    that.setTime();
+  },1000);
+};
+
+var minutes;
+var seconds;
+Chronometer.prototype.setMinutes = function () {
+  minutes = parseInt(this.currentTime/60);
+  return minutes;
+};
+
+Chronometer.prototype.setSeconds = function () {
+  seconds = this.currentTime - parseInt(this.currentTime / 60) * 60;
+  return seconds;
+};
+
+Chronometer.prototype.twoDigitsNumber = function (number) {
+  var twoDigits = '00';
+  if (number < 10){
+    twoDigits = '0' + number;
+  } else if (number === 15) {
+    twoDigits = '15';
+  }
```

Comment:
En este caso no os pide esto! Estáis "hardcodeandolo". Se pide que el numero sea dos dígitos siempre. Una posible solución: 

Chronometer.prototype.twoDigitsNumber = function (value) {
  return ('0' + value).slice(-2);
};

Así siempre será de dos dígitos el número.